debezium-api-generator/src/main/java/io/debezium/api_generator/JsonSchemaCreatorService.java

```diff
@@ -0,0 +1,186 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.api_generator;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.SortedMap;
+import java.util.TreeMap;
+
+import org.apache.kafka.common.config.ConfigDef;
+import org.eclipse.microprofile.openapi.models.media.Schema;
+
+import io.debezium.config.Field;
+import io.debezium.metadata.AbstractConnectorMetadata;
+import io.smallrye.openapi.api.models.media.SchemaImpl;
+
+public class JsonSchemaCreatorService {
+
+    private final String connectorBaseName;
+    private final String connectorName;
+    private final AbstractConnectorMetadata connectorMetadata;
+    private final Set<String> propertyIncludeList;
```

Comment:
@rk3rn3r, apart from the name inconsistency  (list vs set), I didn't figure out how this is sourced, and what's the use case. Can you shed some light? Can we alternatively obtain this from formate or metadata implementations?

Reply:
It was sourced from the `GeneratorCommand` from `buildPropertyIncludeLists()` method. It reads a file from the cli argument: `names = { "-p", "--include-properties-file" }, description = "A file that specifies which properties to include.`

It's basically a file listing one connector property per line that should be included in the schema. It's used to specify which connector properties should be included aka which one should be excluded and so which properties are visible / not visible in the UI.

Reply:
Ok, I see. What's the use case for this? Can we remove this, in favour of having format implementations themselves the ability to apply filters to the properties they want to expose? I think this should do the trick.

Reply:
Agreed on a call to have a method `PropertyFilter ApiFormat::getPropertyFilter()` for letting API formats exclude specific properties. We can reconsider if this proves too unflexible.